plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/catalog/hms/HiveMetastoreTableOperations.java

```diff
@@ -113,10 +113,13 @@ private void commitTableUpdate(Table table, TableMetadata metadata, BiFunction<T
 
             Table updatedTable = tableUpdateFunction.apply(table, newMetadataLocation);
 
+            // Passing environment context causes redundant operations if Hive locking is enabled
```

Comment:
Could you give more details about the consequences that what is "redundant operations" in the commets?